db/migrate/20220825101437_create_cases_table.rb

```diff
@@ -0,0 +1,10 @@
+class CreateCasesTable < ActiveRecord::Migration[7.0]
+  def change
+    create_table :cases, id: :uuid do |t|
+      t.string :urn
+      t.references :crime_application, null: false, foreign_key: true, type: :uuid
```

Comment:
As a precaution let's add `index: { unique: true }` so in case of race conditions it is not possible to end up with 2 case records pointing to the same `crime_application`.